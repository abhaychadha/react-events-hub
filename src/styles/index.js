import { injectGlobal } from "styled-components";
import common from "./common";
import fonts from "./fonts.css";
injectGlobal`  
  ${fonts}
  ${common}
`;
