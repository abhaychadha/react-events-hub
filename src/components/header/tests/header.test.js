import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import Header from '../header';
import styles from '../../../styles/theme';
import { withStyles } from '@material-ui/core/styles';

describe('Header', () => {
  let shallow;
  const StyledHeader = withStyles(styles)(Header);
  beforeEach(() => {
    shallow = createShallow(<StyledHeader/>);
  });

  it('header renders', () => {
    const wrapper = shallow(<StyledHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
