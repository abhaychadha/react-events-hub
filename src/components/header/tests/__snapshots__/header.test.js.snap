// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header header renders 1`] = `
<Header
  classes={
    Object {
      "appBar": "Header-appBar-2",
      "card": "Header-card-9",
      "cardContent": "Header-cardContent-11",
      "cardGrid": "Header-cardGrid-8",
      "cardMedia": "Header-cardMedia-10",
      "heroButtons": "Header-heroButtons-6",
      "heroContent": "Header-heroContent-5",
      "heroUnit": "Header-heroUnit-4",
      "hidden": "Header-hidden-1",
      "icon": "Header-icon-3",
      "layout": "Header-layout-7",
      "typography": "Header-typography-12",
    }
  }
/>
`;
